syntax = "proto3";

package NDTS.NTestingProcessor;

import "docker_container.proto";
import "brocker_client.proto";

message TTestingProcessorConfig {
    TDockerContainerConfig docker_container_config = 1;
    TBrockerClientConfig brocker_client_config = 2;
    string tabasco_url = 3;
    uint64 instance_count = 4;
}
